tests/dataimportcron_test.go

```diff
@@ -367,6 +369,120 @@ var _ = Describe("DataImportCron", func() {
 		Entry("[test_id:8266] succeed deleting error DVs when importing new ones", false, true, 2, cdiv1.DataImportCronSourceFormatPvc),
 	)
 
+	It("[test_id:XXXX] Succeed garbage collecting old version DVs", func() {
```

Comment:
We can delete this test now that we are doing the check in 7406

Reply:
Yup, I was just waiting to see if '7406' is good before deleting the old test